### ISSUE TITLE
Selection filter voice and chord buttons don’t report state to Screen Reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#33898]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

STEPS:
1.  Open a score with notes in at least 2 different voices.
2. Navigate to the selection filter.
3. With screen reader enabled use arrow keys to move through the list of voice buttons and hear each announce "voice 1 button", "voice 2 button", etc.
4. Press enter on any voice.
5. Arrow again across the voice buttons and hear that nothing has changed, none of them report their status as selected or not selected and there was no feedback from the screen reader when enter was pressed on one to say what happened, if anything.
6.  Using a score with some chords select all of one staff (including chords).
7. Navigate back to the selection filter and arrow through the options for chords.  You hear "Top note button", "bottom note button" etc but they do not announce state.
8. Press enter on one of these and again the screen reader gives no indication of what has been selected/filtereds.

It is very hard for a blind user to make any use of this when you don't have feedback to know what you have or have not selected.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.3

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
